Language/Expressions/Throw/evaluation_t04.dart

```diff
@@ -19,6 +19,6 @@ n() => null;
 n2() {}
 
 main() {
-  Expect.throws(() { throw n() ; }, (e) => e is NullThrownError);
-  Expect.throws(() { throw n2(); }, (e) => e is NullThrownError);
+  Expect.throws(() { throw n() ; }, (e) => e is TypeError);
```

Comment:
Drive by comment...

The comment at the top of the file should be updated to match the new expectations.